syz-fuzzer/fuzzer.go

```diff
@@ -385,6 +397,8 @@ func (fuzzer *Fuzzer) addInputToCorpus(p *prog.Prog, sign signal.Signal, sig has
 
 	if !sign.Empty() {
 		fuzzer.signalMu.Lock()
+		fuzzer.sumPrios += int64(len(sign))
+		fuzzer.corpusPrios = append(fuzzer.corpusPrios, fuzzer.sumPrios)
```

Comment:
chooseProgramFromCorpus assumes that corpus and corpusPrios have equal length. But we add to these slices under completely different conditions. We will get out-of-bounds crashes sooner or later.